src/MTGCore.ClientApp/ClientApp/src/App.vue

```diff
@@ -44,138 +45,156 @@
-        data: this.$data
+        data: this.$data,
       })
-          .then(() => {
-            this.updateDeckCardList(this.selectedDeck);
-          })
-          .catch(err => {
-            alert(`There was an error submitting your form. See details: ${err}`);
-          });
+        .then(() => {
+          this.updateDeckCardList(this.selectedDeck);
+        })
+        .catch((err) => {
+          alert(`There was an error submitting your form. See details: ${err}`);
+        });
     },
     SubmitForm() {
       this.searchInProgress = true;
       this.searchedCards = null;
       this.searchedCardsLoadingError = null;
+
+
+      var filter = {}
+      filter["SearchFilter"] = this.searchFilter;
+      filter["SearchFilter"].Name = this.Name;
+
       axios({
-        method: "post",
-        url: "https://localhost:44305/api/Search/",
-        data: this.$data
+        method: 'post',
+        url: 'https://localhost:44305/api/Search/',
+        data: filter,
       })
-          .then(res => {
-            this.searchedCards = res.data;
-          })
-          .catch(err => {
-            this.searchedCardsLoadingError = `Something went wrong: ${err}`;
-          })
-          .finally(() => {
-            this.searchInProgress = false;
-          });
+        .then((res) => {
+          this.searchedCards = res.data;
+        })
+        .catch((err) => {
+          this.searchedCardsLoadingError = `Something went wrong: ${err}`;
+        })
+        .finally(() => {
+          this.searchInProgress = false;
+        });
     },
     updateDeckCardList(id) {
       this.selectedDeck = id;
       axios({
-        method: "get",
+        method: 'get',
         //todo: make this call based on the selected dropdown ID
         url: `https://localhost:44305/api/Deck/${id}`,
-        data: this.$data
+        data: this.$data,
       })
-          .then(res => {
-            this.deckCards = res.data;
-          })
-          .catch(err => {
-            alert(`There was an error submitting your form. See details: ${err}`);
-          });
+        .then((res) => {
+          this.deckCards = res.data;
+        })
+        .catch((err) => {
+          alert(`There was an error submitting your form. See details: ${err}`);
+        });
     },
     updateDeckList() {
       axios({
-        method: "get",
-        url: "https://localhost:44305/api/Deck",
-        data: this.$data
+        method: 'get',
+        url: 'https://localhost:44305/api/Deck',
+        data: this.$data,
       })
-          .then(res => {
-            this.deckList = res.data;
-          })
-          .catch(err => {
-            alert(`There was an error submitting your form. See details: ${err}`);
-          });
-    }
+        .then((res) => {
+          this.deckList = res.data;
+        })
+        .catch((err) => {
+          alert(`There was an error submitting your form. See details: ${err}`);
```

Comment:
Probably an issue for a different PR but we should log here instead of alerting.